docs/source-pytorch/advanced/speed.rst

```diff
@@ -101,26 +101,12 @@ For debugging purposes or for dataloaders that load very small datasets, it is d
 
     warnings.filterwarnings("ignore", category=PossibleUserWarning)
 
-Spawn
```

Comment:
The trainer uses ddp (non-spawn) by default and there is no benefit of using ddp-spawn for speed. If you do use ddp-spawn, the only relevant consideration is the "persistent workers" setting explained in the section below.